sdk/network/arm-network/CHANGELOG.md

```diff
@@ -1,145 +1,11 @@
-  - Operation DscpConfigurationOperations.beginDeleteAndWait has a new signature
-  - Operation DscpConfigurationOperations.get has a new signature
-  - Operation DscpConfigurationOperations.list has a new signature
-  - Operation DscpConfigurationOperations.listAll has a new signature
-  - Operation InboundSecurityRuleOperations.beginCreateOrUpdate has a new signature
-  - Operation InboundSecurityRuleOperations.beginCreateOrUpdateAndWait has a new signature
-  - Operation ServiceTagInformationOperations.list has a new signature
-    
 # Release History
+    
+## 26.0.0 (2021-12-06)
 
-### 26.0.0-beta.1 (2021-10-09)
-
-This is the first preview for the new version of the `@azure/arm-network` package that follows the new [guidelines for TypeScript SDKs](https://azure.github.io/azure-sdk/typescript_introduction.html) for Azure services.
+The package of @azure/arm-network is using our next generation design principles since version 26.0.0, which contains breaking changes.
 
-While this package remains auto generated, the SDK generator itself has undergone changes to comply with the above guidelines in order to generate packages that are idiomatic to the JavaScript/TypeScript ecosystem and consistent with other packages for Azure services. For more on this, please see [State of the Azure SDK 2021](https://devblogs.microsoft.com/azure-sdk/state-of-the-azure-sdk-2021/).
+To understand the detail of the change, please refer to (Changelog)[https://aka.ms/js-track2-changelog]
 
-Please note that this version has breaking changes, all of which were made after careful consideration during the authoring of the guidelines and user studies.
+To migrate the existing applications to the latest version, please refer to (Migration Guide)[https://aka.ms/js-track2-migration-guide]
 
-**Noteworthy changes and features**
-- Authentication: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are no longer supported. Use package [@azure/identity](https://www.npmjs.com/package/@azure/identity) instead. Select a credential from Azure Identity examples based on the authentication method of your choice.
-- Callbacks: Method overloads that used callbacks have been removed and the use of promises is encouraged instead.
-- List operations now return an iterable result that follows the `PagedAsyncIterableIterator` interface as opposed to the previous model where you had to make a new request using the link to the next page.
-- Long running operations i.e. the Lro related object returned by methods whose names started with `begin`, now uses `pollUntilDone` to check whether the request is finished, instead of `pollUntilFinished`. To get the final result, use the corresponding method that will have the suffix `AndWait`.
-- The SDK only supports ECMAScript 2015 (ES6) and beyond, all projects that referenced this SDK should be upgraded to use ES6.
+To learn more, please refer to our documentation (Quick Start)[https://aka.ms/js-track2-quickstart].
```

Comment:
To understand the detail of the change, please refer to [Changelog](https://aka.ms/js-track2-changelog)

To migrate the existing applications to the latest version, please refer to [Migration Guide](https://aka.ms/js-track2-migration-guide)

To learn more, please refer to our documentation [Quick Start](https://aka.ms/js-track2-quickstart).

Reply:
@colawwj This change is included in latesst automation tool. Please change it directly or use the latest tool to re-generate this PR.